OpenRA.Mods.Common/Activities/Air/FlyAttack.cs

```diff
@@ -222,8 +222,14 @@ public FlyAttackRun(Actor self, Target t, WDist exitRange)
 
 		protected override void OnFirstRun(Actor self)
 		{
-			QueueChild(new Fly(self, target, target.CenterPosition));
-			QueueChild(new Fly(self, target, exitRange, WDist.MaxValue, target.CenterPosition));
+			// The target may have died while this activity was queued
```

Comment:
Actually, 
```suggestion
			// The target may have died or changed targetability while this activity was queued
```
However, that is not worth blocking this.